extensions/markdown-language-features/src/languageFeatures/copyFiles/copyPasteLinks.ts

```diff
@@ -34,7 +34,8 @@ class PasteLinkEditProvider implements vscode.DocumentPasteEditProvider {
 		if (!urlList) {
 			return undefined;
 		}
-		const pasteEdit = await createEditAddingLinksForUriList(document, ranges, urlList, token, true);
+		const smartPaste = vscode.workspace.getConfiguration('markdown', document).get<'always' | 'smart' | 'never'>('editor.pasteUrlAsFormattedLink.enabled', 'always');
```

Comment:
Instead of duplicating the call to `getConfiguration`, try creating a new function that both callers can use

Reply:
Also may be worth extracting the string values to an enum